internal/v1/api.yaml

```diff
@@ -591,6 +591,36 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HTTPErrorList'
+  /experimental/blueprints/{id}/imageTypes:
+    parameters:
+      - in: path
+        name: id
+        schema:
+          type: string
+          format: uuid
+        example: '123e4567-e89b-12d3-a456-426655440000'
+        required: true
+        description: UUID of a blueprint
+    get:
+      summary: Get image types from a blueprint
+      description: "get image types from a blueprint"
+      operationId: getImageTypesFromBlueprint
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ImageTypes'
```

Comment:
It is not a good practice to return anything else than a dictionary. Probably okay for this endpoint, it is experimental after all. I am good.